packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -461,6 +461,27 @@ describe("BnsConnection", () => {
     connection.disconnect();
   });
 
+  it("can get a valid header", async () => {
+    pendingWithoutBnsd();
+    const connection = await BnsConnection.establish(bnsdTendermintUrl);
+    const header = await connection.getHeader(3);
+    expect(header.height).toEqual(3);
+    expect(header.appHash.length).toEqual(20);
+  });
+
+  it("throws if it cannot get header", async () => {
+    pendingWithoutBnsd();
+    const connection = await BnsConnection.establish(bnsdTendermintUrl);
+    connection
+      .getHeader(123456)
```

Comment:
This will fail for development chains running 34 hours, which is not uncommon on developer machines. Can you use something significantly larger?